src/centengine/engine_impl.cc

```diff
@@ -263,6 +327,15 @@ grpc::Status engine_impl::GetServiceGroupsCount(
   return grpc::Status::OK;
 }
 
+/**
+ * @brief Return the total number of contact groups.
+ *
+ * @param context gRPC context
+ * @param unused
+ * @param response Map size
+ *
+ * @return Status::OK
+ */
 grpc::Status engine_impl::GetContactGroupsCount(
     grpc::ServerContext* context,
     const ::google::protobuf::Empty* request,
```

Comment:
Could you add __attribute__((unused)) to the request parameter. This will remove possible warnings given during compilation.